aws/modules/sgs/variables.tf

```
@@ -0,0 +1,35 @@
+variable "name" {
+  description = "The name to use for all the cluster resources"
+  type        = string
+}
+
+variable "vpc_id" {
+  description = "The id of the vpc"
+  type        = string
+}
+
+variable "ingress_rules" {
+  description = "List of ingress rules to add"
+  type        = list(list(string))
```

We can add default for ingress and egress rules so that we don't have to specify an empty list if we don't want to specify any
```suggestion
  type        = list(list(string))
  default = []
```